src/components/Review/InitialRatings.js

```diff
@@ -0,0 +1,14 @@
+export const InitialRatings = [
```

Comment:
Only components (and classes in js, but we don't use them) should start with an uppercase letter. Just a general naming convention.

Reply:
also applies to the filename